src/main/java/ch/uzh/ifi/hase/soprafs24/entity/Stats.java

```diff
@@ -16,13 +16,13 @@ public class Stats {
     private Long id;
 
     @NonNull
-    private Integer userId;
+    private Long userId;
```

Comment:
I think we should try to use the Onetomany or Manytoone annotation, this will make querying the stats easier (and is the proper way DB-wise imo). It is somewhat similar to what we used for SpotifyJWT.

--> It's okay to leave that for M4 however.